packages/panzer/mini-em/example/BlockPrec/CMakeLists.txt

```diff
@@ -23,6 +25,18 @@ TRIBITS_COPY_FILES_TO_BINARY_DIR(CopyBlockPrecFiles
   DEST_DIR ${CMAKE_CURRENT_BINARY_DIR}
   )
 
+# rather than list all files like above, I glob all xmls
+file(GLOB ${PACKAGE_NAME}_InputFiles *.xml)
+
+# TODO: ask for comments on how this should be done
+# Tribits does not seem to support installing arbitrary files
+#
+# This will place all xml globbed above into $INSTALL_PREFIX/bin/PanzerMiniEM_InputFiles
+#
+install(FILES ${${PACKAGE_NAME}_InputFiles}
+        DESTINATION bin/${PACKAGE_NAME}_InputFiles/
```

Comment:
@jjellio, actually, as mentioned in:

* https://tribits.org/doc/TribitsBuildReference.html#setting-the-install-prefix

you should not hard-code `bin` but should use `${Project>_INSTALL_RUNTIME_DIR}`.

I guess we need to add a short "HowTo install extra files" to the document:

* https://tribits.org/doc/TribitsDevelopersGuide.html#howtos

Reply:
Actually, it would be better, in my opinion, to install this stuff under `${${PROJECT_NAME}_INSTALL_EXAMPLE_DIR}/Panzer/`

Reply:
The `bin` business was to ensure it matched what Tribits chose. I think the best approach is to use `${${PROJECT_NAME}_INSTALL_EXAMPLE_DIR}/Panzer/`. This does mean I will CMake install the binary + files to that directory, rather than Tribits Install the binary, and CMake install the files. (the distinction being calling cmake's install directly for both, rather than using `INSTALLABLE` with Tribits)

I added `${PACKAGE_NAME}_INSTALL_PREFIX` as a new variable that can override the installation location, and this variable defaults to `${${PROJECT_NAME}_INSTALL_EXAMPLE_DIR}/${PACKAGE_NAME}/`

I am waiting on the build to run, then I can see if actually does what I think it should.

Thanks for the input, I couldn't find a clear guide for this (I actually looked at SEACAS because they install their binaries, they use Cmake install directly)